jit-application/src/main/resources/application.yml

```diff
@@ -85,3 +85,12 @@ logging:
     org:
       hibernate:
         SQL: DEBUG
+
+camel:
```

Comment:
I think this is a nice way specifying the routes. Now I want to refactor mine :)